browser/ui/webui/new_tab_page/brave_new_tab_ui.cc

```diff
@@ -73,6 +79,17 @@ BraveNewTabUI::BraveNewTabUI(content::WebUI* web_ui, const std::string& name)
 
   AddBackgroundColorToSource(source, web_contents);
 
+  // Lottie animations tick on a worker thread and requires the document CSP to
+  // be set to "worker-src blob: 'self';".
+  source->OverrideContentSecurityPolicy(
+      network::mojom::CSPDirectiveName::WorkerSrc,
+      "worker-src blob: chrome://resources 'self';");
+
+  source->OverrideContentSecurityPolicy(
+      network::mojom::CSPDirectiveName::TrustedTypes,
+      "trusted-types static-types lottie-worker-script-loader lit-html-desktop "
```

Comment:
is lit-html-desktop needed? i'm not sure what Lit is or why we need it.

Reply:
Upstream also has this policy for most of webui.
https://source.chromium.org/chromium/chromium/src/+/main:chrome/browser/ui/webui/webui_util.cc;l=112?q=%22lit-html-desktop%22&start=21

Lit is https://lit.dev/. Upstream migrated from `Polymer` to `Lit` recently. cc @petemill

Reply:
Lit is the successor to polymer and chromium is using it for its WebUI. All Chromium WebUI pages have this Trusted Types policy: https://source.chromium.org/chromium/chromium/src/+/main:chrome/browser/ui/webui/webui_util.cc;l=105